cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CombiningOperatorsTest.scala

```diff
@@ -131,11 +131,11 @@ class CombiningOperatorsTest extends DocumentingTest {
       }
     }
     section("AntiSemiApply", "query-plan-anti-semi-apply") {
-      p("""Tests for the existence of a pattern predicate.
-           |`SemiApply` takes a row from its child operator and feeds it to the leaf operator on the right-hand side.
+      p("""Tests for the absence of a pattern predicate.
```

Comment:
This reads weirdly. AntiSemiApply is used as a predicate, that tests for the absence of a matching pattern. Maybe it should read `Tests for the absence of a pattern` - just skip the word predicate